src/num/ops.c

```diff
@@ -1120,15 +1122,18 @@ static DEF_TYPEID(gpu_data_s);
 
 
 #ifdef USE_CUDA
+#ifdef _OPENMP
 #include <omp.h>
 #define MAX_CUDA_DEVICES 16
 omp_lock_t gpulock[MAX_CUDA_DEVICES];
 #endif
+#endif
 
 
 static void gpuwrp_fun(const operator_data_t* _data, unsigned int N, void* args[N])
 {
 #ifdef USE_CUDA
+#ifdef _OPENMP
 	const struct operator_s* op = CAST_DOWN(gpu_data_s, _data)->op;
 	void* gpu_ptr[N];
```

Comment:
This seems wrong, or? Now it fails silently without CUDA and fails with an assertion without _OPENMP while it should fail with an assertion without CUDA and work without _OPENMP (or many also fail but not silently).